app/generator/content/schedule/part1/index.js

```diff
@@ -11,11 +11,10 @@ const part1 = (schedule) => {
     stack: [
       { image: `${imagePath}/rpa-logo.jfif`, fit: [millimetresToPoints(200), millimetresToPoints(25)], style: 'scheduleLogo' },
       getAddress(schedule.businessName, schedule.address),
-      { text: `${schedule.scheme.name}`, style: 'header1' },
-      { text: 'Revised payment schedule', style: 'subTitle' },
-      getBusinessName(schedule.businessName),
       getSBI(schedule.sbi),
```

Comment:
This displays wrong title should be `SBI:` but is `Single business identifier (SBI):`

Also, we have 2 of the same content for both statement (`app/generator/content/statement/summary/get-sbi.js`) and schedule (`app/generator/content/schedule/part1/get-sbi.js`), can we move this into the common area like `app/generator/content/get-business-name.js` and reference it in both cases.

Reply:
Sorry I didn't add update, the files and expected result are not the same, hence, better to keep them apart as they are.